packages/dds/map/src/directory.ts

```diff
@@ -192,24 +192,45 @@ export type IDirectoryOperation = IDirectoryStorageOperation | IDirectorySubDire
 
 /**
  * Defines the in-memory object structure to be used for the conversion to/from serialized.
- * @privateRemarks
- * Directly used in JSON.stringify, direct result from JSON.parse.
+ *
+ * @remarks Directly used in
+ * {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify
+ * | JSON.stringify}, direct result from
+ * {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse | JSON.parse}.
  */
 export interface IDirectoryDataObject {
+    /**
+     * TODO
```

Comment:
Any recommendations on docs for this? Some other definition we should point to?

Reply:
These would represent the same kind of key/value pairing as `IMapDataObjectSerializable` (they are the key/values actually stored by the user, as opposed to the subdirectories).  I don't know that they should necessarily reference each other but should have at least very similar explanation.